src/patternfly/components/CodeEditor/code-editor.scss

```diff
@@ -0,0 +1,91 @@
+
+  // code
+  --pf-c-code-editor__code--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-code-editor__code--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-code-editor__code--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  // tab
+  --pf-c-code-editor__tab--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-code-editor__tab--Color: var(--pf-global--Color--200);
+  --pf-c-code-editor__tab--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-code-editor__tab--Top: var(--pf-c-code-editor__code--before--BorderWidth);
+  --pf-c-code-editor__tab--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-code-editor__tab--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-code-editor__tab--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-code-editor__tab--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-code-editor__tab--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-code-editor__tab--before--BorderTopColor: var(--pf-global--BorderColor--100);
+  --pf-c-code-editor__tab--before--BorderRightColor: var(--pf-global--BorderColor--100);
+  --pf-c-code-editor__tab--before--BorderBottomColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-code-editor__tab--before--BorderLeftColor: var(--pf-global--BorderColor--100);
+
+  width: 100%;
+}
+
+.pf-c-code-editor__control {
+  position: relative;
+  height: var(--pf-c-code-editor__control--Height);
+}
+
+.pf-c-code-editor__action-group {
+  position: absolute;
+  z-index: var(--pf-c-code-editor__action-group--ZIndex);
+  display: flex;
+
+  .pf-c-button.pf-m-control {
+    --pf-c-button--m-control--Color: var(--pf-c-code-editor__action-group--c-button--m-control--Color);
+
+    &:hover {
+      --pf-c-code-editor__action-group--c-button--m-control--Color: var(--pf-c-code-editor__action-group--c-button--m-control--hover--Color);
+    }
+
+    &:focus {
+      --pf-c-code-editor__action-group--c-button--m-control--Color: var(--pf-c-code-editor__action-group--c-button--m-control--focus--Color);
+    }
+  }
+}
+
+.pf-c-code-editor__code {
+  position: relative;
+  background-color: var(--pf-c-code-editor__code--BackgroundColor);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-code-editor__code--before--BorderWidth) solid;
+    border-color: var(--pf-c-code-editor__code--before--BorderColor);
+  }
+}
+
+.pf-c-code-editor__tab {
+  position: absolute;
+  right: 0;
+  z-index: var(--pf-c-code-editor__tab--ZIndex);
+  padding: var(--pf-c-code-editor__tab--PaddingTop) var(--pf-c-code-editor__tab--PaddingRight) var(--pf-c-code-editor__tab--PaddingBottom) var(--pf-c-code-editor__tab--PaddingLeft);
+  color: var(--pf-c-code-editor__tab--Color);
+  background-color: var(--pf-c-code-editor__tab--BackgroundColor);
+
+  &::before {
```

Comment:
you could use a border here instead of a pseudo element, too. This pseudo element is keeping you from being able to click on the content in the tab.